src/v2i-hub/CDA1TenthPlugin/src/CDA1TenthPlugin.cpp

```diff
@@ -59,8 +60,8 @@ void CDA1TenthPlugin::UpdateConfigSettings() {
 	GetConfigValue<bool>("Webservice_Secure", secure);
 	// Polling Frequency in seconds
 	GetConfigValue<uint16_t>("Webservice_Polling_Frequency", polling_frequency);
-
-	client = std::make_shared<WebServiceClient>( host, port, secure, polling_frequency );
+	//TODO: Commented out due to compilation error
```

Comment:
Why is there a compilation error when we attempt to send messages to web service? Please also explain why we do not plan to address this in the current story or Pull Request